src/table/operations.jl

```diff
@@ -0,0 +1,85 @@
+import Base: filter, map, reduce
+
+"""
+    function map(f, d::DTableRows)
+
+Applies `f` to each row of the `DTable`.
+
+# Examples
+```jldoctest        
+    d = DTable((a=rand(1000), b=rand(1000));chunksize=100)
+    m = map(x -> x.a + x.b, eachrow(d))
+    fetch(m)
+```
+"""
+function map(f, d::DTableRows)
+    chunk_wrap = chunk -> begin 
+        i = eachrow(chunk)
+        map(f, i)    
+    end
+    result_vector = [Dagger.@spawn chunk_wrap(c) for c in d.dtable.chunks]
+    Dagger.@spawn ((r...) -> vcat((r)...))(result_vector...)
+end
+
+function map(f, d::DTableColumns)
+    throw("Not implemented")
```

Comment:
I left it out intentionally for now. I don't ever do these kinds of operations, but I know it's a thing, so I just put a placeholder. Seems to me that implementation of this might get more complicated. I'll look into it at some point, but not in this PR